experimental/packages/opentelemetry-api-metrics/src/types/Metric.ts

```diff
@@ -72,27 +72,27 @@ export interface Counter {
   /**
    * Increment value of counter by the input. Inputs may not be negative.
    */
-  add(value: number, attributes?: Attributes, context?: Context): void;
+  add(value: number, attributes?: SpanAttributes, context?: Context): void;
 }
 
 export interface UpDownCounter {
   /**
    * Increment value of counter by the input. Inputs may be negative.
    */
-  add(value: number, attributes?: Attributes, context?: Context): void;
+  add(value: number, attributes?: SpanAttributes, context?: Context): void;
 }
 
 export interface Histogram {
   /**
    * Records a measurement. Value of the measurement must not be negative.
    */
-  record(value: number, attributes?: Attributes, context?: Context): void;
+  record(value: number, attributes?: SpanAttributes, context?: Context): void;
 }
 
 /**
  * key-value pairs passed by the user.
  */
-export type Attributes = { [key: string]: string };
+export type SpanAttributes = { [key: string]: string };
```

Comment:
```suggestion
export type MetricAttributes = { [key: string]: string };
```

Reply:
This applies to several places in this file and most likely impacts other metrics packages where you moved from `Attribute` to `SpanAttribute`

Reply:
yeah it does